dependencies.gradle

```diff
@@ -5,8 +5,7 @@ ext {
   retroLambdaPatch = 'me.tatarka.retrolambda.projectlombok:lombok.ast:0.2.3.a2'
   compileSdkVersion = 25
   buildToolsVersion = '25.0.1'
-  minSdkVersionDebug = 21
-  minSdkVersion = 17
+  minSdkVersion = 16
```

Comment:
API 16 doesn't support RTL features (such as `marginStart`, `marginEnd`, and so on)
for supporting API 16, we should replace them with app-compat expressions (such as `android:layout_marginStart` to `app:layout_marginStart` )